ci/common/suite.sh

```diff
@@ -45,23 +45,23 @@ fail() {
 
 run_test() {
   local cmd="$1"
-  shift
+  test $# -gt 0 && shift
   local test_name="$1"
   : ${test_name:=$cmd}
-  shift
+  test $# -gt 0 && shift
   if ! eval "$cmd" ; then
     fail "${test_name}" "$@"
   fi
 }
 
 run_test_wd() {
   local timeout="$1"
-  shift
+  test $# -gt 0 && shift
   local cmd="$1"
-  shift
+  test $# -gt 0 && shift
   local test_name="$1"
```

Comment:
`$2`, etc cannot be used because arguments used by this function must be got rid of so that `$@` passed to `fail` will have expected arguments.